packages/editor-ui/src/components/RunData.vue

```diff
@@ -760,7 +760,7 @@ export default defineComponent({
 	computed: {
 		...mapStores(useNodeTypesStore, useNDVStore, useWorkflowsStore, useSourceControlStore),
 		isReadOnlyRoute() {
-			return this.$route?.meta?.readonly === true;
+			return this.$route?.meta?.readOnlyCanvas === true;
```

Comment:
[created follow up to stop using this directly here](https://linear.app/n8n/issue/ADO-1646/tech-debt-remove-readonlycanvas-from-inner-compoennts)